### ISSUE TITLE
localization: "Message ... didn't get externalized correctly"

### ISSUE THREAD

**Issue by justinmk3** [aws/aws-toolkit-vscode#1774]
## Problem

vscode extension host sometimes reports warnings of the form:

> Message ... didn't get externalized correctly.

## Solution?

Haven't confirmed, but based on https://github.com/microsoft/vscode/issues/42426 / https://github.com/microsoft/vscode-node-debug/commit/4d9a3a70fb776e05df218849a68cdfd2645d4ed0 we may need to do:

1. call `nls.loadMessageBundle()` directly in modules instead of using `src/shared/utilities/vsCodeUtils.ts:localize()`
2. remove `src/shared/utilities/vsCodeUtils.ts:localize()`

**Comment by JadenSimon** [comment]
> call nls.loadMessageBundle() directly in modules instead
Yup this is a requirement for NLS. Indirect calls would result in the file looking for a localization file named after the imported module/file. But the `localize` calls would still be extracted using the current module name, so the module never finds anything.
